unit-tests/native/src/test/scala/org/scalanative/testsuite/posixlib/TermiosTest.scala

```diff
@@ -0,0 +1,27 @@
+package org.scalanative.testsuite.posixlib
+
+import org.junit.Assert._
+import org.junit.Test
+
+import scala.scalanative.unsafe._
+import scala.scalanative.unsigned._
+import scalanative.meta.LinktimeInfo.isWindows
+import scalanative.posix.termios
+
+class TermiosTest {
+
+  def setRawMode(): Unit = {
+    Zone.acquire { implicit z =>
+      val attrs = alloc[termios.termios]()
+      termios.tcgetattr(1, attrs)
+      attrs._4 = attrs._4 & ~(termios.ECHO | termios.ICANON).toUInt
+      termios.tcsetattr(1, termios.TCSAFLUSH, attrs)
+    }
+  }
+
+  @Test def testSetRawMode(): Unit = if (!isWindows) {
```

Comment:
Understod: This rapid prototyping draft

At the risk of saying what you probably intended once
you got past step 1.

Down the line, probably want to fetch the attributes
just set, into a new poisoned (known pattern, such as repeated DEADBEEF )buffer & compare the two.

Of course any mismatch leaves open the question
of "is it set or get that is broken"?  We all are
familiar with that one.  At least the mismatch is 
detected.

Reply:
This is the use case given in the ticket. It failed when running that code. https://github.com/scala-native/scala-native/issues/4143